src/main/java/org/hmhb/user/UserDao.java

```diff
@@ -17,4 +19,11 @@
      */
     HmhbUser findByEmailIgnoreCase(String email);
 
+    /**
+     * Returns all {@link HmhbUser}s ordered ascending by their emails.
+     *
+     * @return all {@link HmhbUser}s
+     */
+    List<HmhbUser> findAllByOrderByEmailAsc();
```

Comment:
Would we not want to order by display name, then by email?


* We would show in a 3-line list view the display name as the first line, the email as the second line, and the role (Super Admin, Admin, User) as the third line.
* Users might know names better than email addresses.
* If someone has multiple google accounts, this would (maybe) list them together (if display name the same) so that this could be discovered.

Example of a list item:
```
John Doe
jdoenuts2@example.org
Super admin
```